frontend/apps/app/components/FormatIcon/icons/SchemaRbIcon.tsx

```diff
@@ -1,13 +1,16 @@
+import Image from 'next/image'
 import type { FC } from 'react'
+import schemaRbIcon from '@/public/assets/schema-rb-icon.png'
 import type { IconProps } from './types'
 
 export const SchemaRbIcon: FC<IconProps> = ({ size = 16 }) => {
   return (
-    <img
-      src="/assets/schema-rb-icon.png"
-      alt="SchemaRbIcon"
+    <Image
+      src={schemaRbIcon}
+      alt="SchemaRb"
```

Comment:
Since this is the text that users will see, the following looks good:

```suggestion
      alt="schema.rb icon"
```